newsletter/_posts/2019-04-01-Jlab-Workshop.md

```diff
@@ -0,0 +1,122 @@
+---
+title: HSF-OSG-WLCG Workshop at Jefferson Lab, HOW2019
+author: Graeme Stewart
+layout: newsletter
+---
+
+The annual [HEP Software Foundation
+workshop](https://indico.cern.ch/event/759388/){:target="jlab_agenda"} happened
+this year in [Jefferson
```

Comment:
Remove 'this year' as it is rather obvious and is being repeated at the beginning of next sentence.